Files/Interacts/Interaction.cs

```diff
@@ -427,8 +427,174 @@ public async void OpenFileLocation_Click(object sender, RoutedEventArgs e)
                 await DialogDisplayHelper.ShowDialogAsync("InvalidItemDialogTitle".GetLocalized(),
                     string.Format("InvalidItemDialogContent".GetLocalized()), Environment.NewLine, destFolder.ErrorCode.ToString());
             }
+        }/// <summary>
+         /// Navigates to a directory or opens file
+         /// </summary>
+         /// <param name="path"></param>
+         /// <param name="itemType"></param>
+         /// <param name="openSilent">Determines whether history of opened item is saved (... to Recent Items/Windows Timeline)</param>
+        public async Task<bool> OpenPath(string path, FilesystemItemType? itemType = null, bool openSilent = false)
+        // TODO: This Function is not ready yet! It is not fully merged from OpenSelectedItems() because of ListedItem restriction, halp needed plz
+        // TODO: This function reliability has not been extensively tested
+        // TODO: Split this function to call OpenFile() and OpenDirectory() separately
+        {
+            bool isHiddenItem = false; // TODO: Determine
```

Comment:
Can be done like this if needed: `IsHidden = NativeFileOperationsHelper.HasFileAttribute(ItemPath, System.IO.FileAttributes.Hidden);`

Reply:
> There's quite a lot of duplicate code, between OpenPath and OpenSelectedItems. I hope you can do something about that xD

I know, I was originally planning to get rid of the `OpenSelectedItems()` function for much better `OpenPath()` - but as you can see, `OpenPath()` is not completed - check for hidden items, shortcut items are not present in `OpenPath()` and I don't know how to implement it without `ListedItem`